templates/seL4VMParameters.template.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2023, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <camkes.h>
+
+/*- if configuration[me.name].get('vm_address_config') -*/
```

Comment:
It's curious that this isn't always set because of the default attribute values.

Reply:
Yeah I wondered about that. The camkes structure was defined in the header file, but the actual struct wasn't in the source.